ajax/libs/videojs-hotkeys/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "videojs-hotkeys",
+  "version": "0.2.20",
+  "description": "Adds more hotkey support to video.js",
+  "filename": "videojs.hotkeys.min.js",
+  "author": "Chris Dougherty",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ctd1500/videojs-hotkeys.git"
+  },
+  "keywords": [
+    "videojs",
+    "videojs-plugin",
+    "hotkeys"
```

Comment:
@sashberd Maybe we can ad more keywords, such as `video`.
What do you think?
I refer it from https://github.com/ctd1500/videojs-hotkeys/blob/master/bower.json#L18
Thank you.